icc-x-api/icc-crypto-x-api.ts

```diff
@@ -1334,6 +1336,14 @@ export class IccCryptoXApi {
     )
   }
 
+  // noinspection JSUnusedGlobalSymbols
+  saveKeychainValidityDateInBrowserLocalStorage(id: string, date: number) {
+    localStorage.setItem(
+      this.keychainValidityDateLocalStoreIdPrefix + id,
+      date.toString() || undefined
```

Comment:
Setting it to `undefined` will actually set it to `"undefined"`. Therefore the value you put in is not the same as the one you get out. This can be a problem when you compare the stored value with an incoming one, as is done below.

Reply:
@cebamps Ok, I though the setItem method would handle change it as an empty string itself... I was wrong :-)